src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,63 @@
+import { useParams } from 'react-router-dom';
+import classnames from 'classnames';
+import { Person } from '../../types';
+import { getFatherPerson, getMotherPerson } from '../../utils/utils';
+import { PersonLink } from '../PersonLink';
+import { tableColumnNames } from '../../utils/constants';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {tableColumnNames.map(name => <th key={name}>{name}</th>)}
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            sex,
+            born,
+            died,
+            fatherName,
+            motherName,
+            slug,
+          } = person;
+
+          return (
+            <tr
+              data-cy="person"
+              className={classnames({
+                'has-background-warning': slug === personSlug,
+              })}
+            >
+              <td>
+                <PersonLink person={person} />
+              </td>
+
+              <td>{sex}</td>
+              <td>{born}</td>
+              <td>{died}</td>
+              <td>
+                {getMotherPerson(people, motherName)}
```

Comment:
Better not to use functions inside JSX. Better to refactor your solution. You can prepare your people array and add `father` and `mother` to every person, and do all other logic right here